go/vt/schema/name_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 The Vitess Authors.
```

Comment:
this is a new file, right? Copyright should say 2020

Reply:
fixed